### DISCUSSION TITLE
Tried to Abort Plotting Process Shortly After Starting. Chia Unresponsive.

### DISCUSSION THREAD

**Discussion by Hated-Humanist** [Chia-Network/chia-blockchain#2018] [Q&A]
I was playing around with Chia 1.05 and some drives I had and tried to make a new plot. I attempted to abort the new plot using the GUI in Windows only to get stuck in a neverending "connecting to wallet..." loop. I figured I could uinstall and reinstall, didn't see an executable for uninstall, and so attempted to deleted installed filed manually and clean the registry.

Anyway, Chia is now bricked for me and whenever I open the installer it just brings me back to the "connecting to wallet" window and continues to utilize my CPU and RAM in the background. I spent a lot of time looking for an answer so I apologize if this is redundant. Can anyone help fill me in about how to start over fresh with Chia 1.05? Much appreciated!

**Comment by hoffmang9** [comment]
It's a known problem. We will temporarily be removing delete plot from 1.1 until it is fixed.
